diesel/src/pg/types/money.rs

```diff
@@ -39,6 +40,40 @@ impl ToSql<types::Money, Pg> for PgMoney {
     }
 }
 
+/// # Panics
+/// will `panic!` on overflow
```

Comment:
I think we should specify "including release mode", since that's where this differs from the default.